### PR TITLE
Making all cards of equal length and resolving overflowing of circle image

### LINKED ISSUE #12: [Improvements] In the About Us, Testimonials and Team Section
## Here are Two Issues that I would like to work on:-
*Issue -1  The Height of all cards is not the same in the Team and Testimonials section. We can resolve this issue to bring consistency to the page.*
*Issue -2 The Circle image is getting overflow from its respective div.*

**Please assign me these issues under Gssoc'23 @varun-singhh**

vovoca
vovoca 3
vovoca 2

### PR DESCRIPTION
**Pull request by Anshika-Jain02**
## Issue #12  resolved
*The height of various placards in the testimonial and team sections was not the same and it was giving the whole website a disoriented look and now it is looking fine.*

## Before :-
vovoca 3
vovoca

## After:-
vovoca_PR2
vovoca_PR4

*The circle image is overflowing out of its div and now, I have corrected this issue*
## Before :-
vovoca 2

## After:-
vovoca_PR3

**Thank you sir for giving me this opportunity**

### CODE REVIEW

**Comment by Anshika-Jain02** [comment]
@varun-singhh please review these changes.

**Comment by varun-singhh** [comment]
@Anshika-Jain02 resolve merge conflicts

**Comment by Anshika-Jain02** [comment]
@varun-singhh sir, can you please guide me a little on how to resolve the merge conflict as I am new to this, and hence, I have no idea.

**Comment by varun-singhh** [comment]
> @varun-singhh sir, can you please guide me a little on how to resolve the merge conflict as I am new to this, and hence, I have no idea.

take a pull from development branch and merge it to you branch `VOVOCA-12`

File: styles/Testimonial.module.css [hunk 1]
```diff
@@ -46,7 +47,12 @@
...
   display: flex;
 }
-
+.developers_article{
+  height: 22rem;
+}
+.contributors_article{
+  height: 7.5rem;
+}
 .testimonial__img {
```
Comment (varun-singhh) [inline styles/Testimonial.module.css:56]: can you check for size in mobile view and attach the screen shot for the same

**Review by varun-singhh — CHANGES_REQUESTED** [review]

**Comment by Anshika-Jain02** [comment]
@varun-singhh please check whether the merge conflicts are resolved or not. Also, as the same class name "testimonial article" is used in all placards, changing its dimensions has affected some other containers like the contributors_article. so, I added this class to resolve the issue in all placards for big screen sizes.

You can check the size for this in mobile view by the below screenshot and #29 is taking care of its size in the mobile view accordingly.

vovoca_changes_PR

**Review by varun-singhh — APPROVED** [review]

**Comment by Anshika-Jain02** [comment]
@varun-singhh please assign me appropriate level tag for this PR

**Pull request merged**